apps/backend/src/db/migrations/20240906080253_add_allow_notifications_column_to_user_details.ts

```diff
@@ -0,0 +1,26 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	USER_DETAILS: "user_details",
+} as const;
+
+const ColumnName = {
+	ALLOW_NOTIFICATIONS: "allow_notifications",
+} as const;
+
+function up(knex: Knex): Promise<void> {
+	return knex.schema.alterTable(TableName.USER_DETAILS, (table) => {
+		table
+			.string(ColumnName.ALLOW_NOTIFICATIONS)
```

Comment:
Can we store boolean, instead of string?

Reply:
I made type: string, because in the future there will be more answers and I will have to change the type again. Also, input accepts a string in option.value, so I made this type for consistency.